Backend/app/rider-platform/rider-app-drainer/src/Types/DBSync/Create.hs

```diff
@@ -30,8 +32,7 @@ import qualified "rider-app" Storage.Beam.Merchant.MerchantServiceConfig as Merc
 import qualified "rider-app" Storage.Beam.Merchant.MerchantServiceUsageConfig as MerchantServiceUsageConfig
 import qualified "rider-app" Storage.Beam.MerchantConfig as MerchantConfig
 import qualified "rider-app" Storage.Beam.OnSearchEvent as OnSearchEvent
-import qualified "rider-app" Storage.Beam.Payment.PaymentOrder as PaymentOrder
-import qualified "rider-app" Storage.Beam.Payment.PaymentTransaction as PaymentTransaction
```

Comment:
These two modules not used anywhere except drainer. I think they was created by mistake and should be removed